src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -13,22 +13,23 @@ class CrudField {
             this.input = this.wrapper.closest("[bp-field-main-input");
             // if no bp-field-main-input has been declared in the field itself,
             // assume it's the first input in that wrapper, whatever it is
-            if (this.input.length == 0) {
+            if (this.input.length === 0) {
                 this.input = $('[bp-field-name="'+ this.name +'"] input, [bp-field-name="'+ this.name +'"] textarea, [bp-field-name="'+ this.name +'"] select').first();
             }
-            this.value = this.input.val();
```

Comment:
Regarding this, the value is set in the beginning, and if it changes, `this.value` won't get updated. So this may induce developers to think they are accessing the correct/current value, and they may not.

Reply:
Agreed. Let's do an accessor for the `value` instead of a property.

Reply:
Accessor added 👌